frontend/__mocks__/crds.ts

```diff
@@ -68,6 +75,9 @@ export const testCRDVersionV1: CRDVersion = {
   name: 'v1',
   storage: true,
   served: true,
+  schema: {
```

Comment:
Not sure if `schema` is optional or not (I specified it is it [types.ts](https://github.com/openshift/console/pull/6302/files#diff-26cde32482ccee5968a1f4d2d69ffe03R486)).  @TheRealJon or @spadgett, do you know?

Reply:
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#customresourcedefinitionversion-v1-apiextensions-k8s-io sure would be helpful if it specified.

Reply:
I believe `schema` is required, so I've changed it.